solutions/aspnet/Makefile

```diff
@@ -8,6 +8,8 @@ APPNAME=webapp
 #OPTS += --strace
 #endif
 
+OPTS = --ram=256m
```

Comment:
This setting should be shadowed by the one in config.json. Does it make a difference for you?

Reply:
Good catch. I change the name in the config and docs to HeapSize but continued to support for UserMemSize for backwards compatibility.